requirements.in

```diff
@@ -25,7 +25,7 @@ drf-extensions==0.7.1
 django-reversion==4.0.1
 django-storages==1.11.1
 djoser==2.1.0
-edx-api-client==1.4.1
+git+https://github.com/mitodl/edx-api-client@4d86e512532d17f15f5f79307b6c254e65c3e409#egg=edx-api-client
```

Comment:
This and the other change in requirements.txt will be replaced with an actual package pin when we publish a new edx-api-client package with https://github.com/mitodl/edx-api-client/pull/93 merged.